assets/app/views/builds.html

```diff
@@ -8,21 +8,86 @@
         </div>
       </div>
-      <div style="margin-bottom: 10px;" ng-repeat="build in builds">
-        <h3>{{build.metadata.labels.buildconfig}} <span class="small">({{build.metadata.name}})</span></h3>
-        <div>Created: <relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp></div>            
-        <div>Status: {{build.status}}</div>
-        <div>Build strategy: {{build.parameters.strategy.type}}</div>
-        <div ng-if="build.parameters.strategy.type == 'STI'">
-          Builder image: {{build.parameters.strategy.stiStrategy.builderImage}}
+      <div ng-repeat="(buildConfigName, buildConfig) in buildConfigs">
+        <h2>{{buildConfigName}}</h2>
+        <div>Build strategy: {{buildConfig.parameters.strategy.type}}</div>
+        <div ng-if="buildConfig.parameters.strategy.type == 'STI'">
+          Builder image: {{buildConfig.parameters.strategy.stiStrategy.builderImage}}
         </div>
-        <div ng-if="build.parameters.source">
-          <div ng-if="build.parameters.source.type == 'Git'">Source repo: {{build.parameters.source.git.uri}}</div>
-          <!-- TODO add git ref details -->
+        <div ng-if="buildConfig.parameters.source">
+          <div ng-if="buildConfig.parameters.source.type == 'Git'">Source repo: {{buildConfig.parameters.source.git.uri}}</div>
         </div>
-        <div>Output image: {{build.parameters.output.imageTag}}</div>   
-        <div>Output image registry: {{build.parameters.output.registry}}</div>   
-      </div>      
+        <div ng-switch="buildConfig.parameters.output.to.kind">
+          <div ng-switch-when="ImageRepository">
+            Output image: {{buildConfig.parameters.output.to.name}}
+          </div>
+        </div>
+        <div ng-if="buildConfig.triggers.length > 0">
+          Triggers: 
+          <ul>
+            <li ng-repeat="trigger in buildConfig.triggers">
+              <span ng-switch="trigger.type">
+                <span ng-switch-when="github">
+                  GitHub webhook URL: <input class="small" readonly="readonly" value="{{buildConfigName | webhookURL : trigger.type : trigger.github.secret : project.metadata.name}}" select-on-focus />
+                  <a href="http://docs.openshift.org/latest/using_openshift/builds.html#webhook-triggers" target="_blank"><i class="pficon pficon-help"></i></a>
+                </span>
+                <span ng-switch-when="generic">
+                  Generic webhook URL: <input class="small" readonly="readonly" value="{{buildConfigName | webhookURL : trigger.type : trigger.generic.secret : project.metadata.name}}" select-on-focus />
+                  <a href="http://docs.openshift.org/latest/using_openshift/builds.html#webhook-triggers" target="_blank"><i class="pficon pficon-help"></i></a>
+                </span>
+                <span ng-switch-when="imageChange">
+                  Image change from image {{trigger.imageChange.image}}:{{trigger.imageChange.tag}}
```

Comment:
in fact you really only have it pointing to an imagerepo.  the "image" field in an ICT tells us what value search/replace in the BuildConfig, it is not telling you the actual upstream image we're monitoring for changes.  So we should definitely show the reference to the ImageRepository instead, here (plus the tag, which is correct)